### ISSUE TITLE
In Linking, incorrect case on two functions

### ISSUE THREAD

**Issue by pillowsoft** [necolas/react-native-web#339]
In: https://github.com/necolas/react-native-web/blob/master/src/apis/Linking/index.js

Incorrect casing on two functions. They should match react-native, thus:

getInitialUrl should be  -> getInitialURL
canOpenUrl should be  -> canOpenURL

**Comment by pillowsoft** [comment]
Looks like **getInitialUrl** needs to return a promise for a url rather than a url, like this:

```
getInitialURL:function getInitialURL() {
  return new Promise(function (resolve, reject) { resolve(''); });
},
```
else you will get an exception that **getInitialURL** has no **then** method.

**Comment by pillowsoft** [comment]
Or rather:

```
  getInitialURL() {
    return new Promise(function (resolve, reject) { resolve(''); });
  },
```

I was looking at the dist rather than the src

**Comment by RangerMauve** [comment]
@pillowsoft Instead of using a promise constructor, you can just use `Promise.resolve(theValue)`

**Comment by necolas** [comment]
Should be fixed by 1d919787d6dd96fe8606220b85e87f84de6a47b6
